xml/System.Text.Json/Utf8JsonWriter.xml

```diff
@@ -617,13 +583,9 @@ Call <xref:System.Text.Json.Utf8JsonWriter.Flush%2A> before getting the current
         <param name="value">The value to be written as a JSON number as part of the name/value pair.</param>
         <param name="escape"><see langword="false" /> to indicate the writer should assume the property name is properly escaped and skips the escaping step; otherwise, <see langword="true" />. This is an optional parameter; its default value is <see langword="true" />.</param>
         <summary>Writes the property name and <see cref="T:System.Double" /> value (as a JSON number) as part of a name/value pair of a JSON object.</summary>
-        <remarks>
-            Writes the <see cref="T:System.Double" /> using the default <see cref="T:System.Buffers.StandardFormat" /> (that is, 'G').
-            </remarks>
-        <exception cref="T:System.ArgumentException">
-            <paramref name="utf8PropertyName" /> is too large.</exception>
-        <exception cref="T:System.InvalidOperationException">
-            The JSON data to be written would be invalid JSON (while validation is enabled).</exception>
+        <remarks>Writes the <see cref="T:System.Double" /> using the default <see cref="T:System.Buffers.StandardFormat" /> (that is, 'G').</remarks>
+        <exception cref="T:System.ArgumentException"><paramref name="utf8PropertyName" /> is too large.</exception>
```

Comment:
Looks like the formatting of exception needs to be tweaked.
The exception "attribute" is on the first line, the text itself is on the next line.

I am not sure if this is important.

cc @mairaw

Reply:
I am aware of this issue. I can't do much about it. I can either save an XDocument with endlines after tags and tabbing, or none of those (all the text in one line).
The problem happens when XElement value begins with a &lt;tag&gt;, even if it's not a real true child, but simply a tag inside plain text. When the value starts with plain text, no endlines are added.
I'll ask ithe XML area owner if this is a bug.
I corrected the issue in my latest commit. I think this comment is outdated now.